_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -154,79 +154,79 @@ Some user agents treat the value of `aria-*` attribute as case-sensitive (even w
 
 #### Failed Example 1
 
-`aria-required` property with invalid true/false value
+This `div` element has an `aria-required` property with invalid `true/false` value.
 
 ```html
 <div role="textbox" aria-required="undefined" aria-label="A required textbox"></div>
 ```
 
 #### Failed Example 2
 
-`aria-expanded` state with invalid true/false/undefined value
+This `div` element has an `aria-expanded` state with invalid `true/false/undefined` value.
 
 ```html
 <div role="button" aria-expanded="mixed">A button</div>
 ```
 
 #### Failed Example 3
 
-`aria-pressed` state with invalid tristate value
+This `div` element has an `aria-pressed` state with invalid tristate value.
 
 ```html
 <div role="button" aria-pressed="horizontal">An other button</div>
 ```
 
 #### Failed Example 4
 
-`aria-errormessage` property with invalid ID reference value, since space is not allowed in a single ID
+This `div` element has an `aria-errormessage` property with invalid ID reference value, since space is not allowed in a single ID.
 
 ```html
 <div role="textbox" aria-errormessage="error1 error2" aria-label="A textbox with an error"></div>
 ```
 
 #### Failed Example 5
 
-`aria-rowindex` property with invalid integer value
+This `div` element has an `aria-rowindex` property with invalid integer value.
 
 ```html
 <div role="gridcell" aria-rowindex="2.5">Fred</div>
 ```
 
 #### Failed Example 6
 
-`aria-valuemin`, `aria-valuemax` and `aria-valuenow` property with invalid number values
+This `div` element has an `aria-valuemin`, `aria-valuemax` and `aria-valuenow` property with invalid number values.
 
 ```html
 <div role="spinbutton" aria-valuemin="one" aria-valuemax="three" aria-valuenow="two" aria-label="Choose a value"></div>
 ```
 
 #### Failed Example 7
 
-`aria-live` property with invalid token value
+This `div` element has an `aria-live` property with invalid token value.
 
 ```html
 <div role="main" aria-live="nope"></div>
 ```
 
 #### Failed Example 8
 
-Element with invalid token list value
+This `div` element has an Element with invalid token list value.
 
 ```html
 <div role="dialog" aria-dropeffect="invalid move"></div>
 ```
 
 #### Failed Example 9
 
-`aria-expanded` state with invalid true/false/undefined value for custom element
+This `div` element has an `aria-expanded` state with invalid `true/false/`undefined` value for custom element.
 
 ```html
 <my-button role="button" aria-expanded="collapsed">My button</my-button>
 ```
 
 #### Failed Example 10
 
-`aria-controls`, which is a required property for the role `scrollbar`, references an element that does not exist in the same document tree.
+This `div` element has an `aria-controls`, which is a required property for the role `scrollbar`, references an element that does not exist in the same document tree.
```

Comment:
Same as Passed Example 10. Especially here since some of the attributes are OK and some aren't.